vcpkg/reference/vcpkg-json.md

```diff
@@ -281,7 +303,7 @@ Each dependency is a string or an object with the following fields:
 | Name | Required | Type   | Description |
 |------|----------|--------|-------------|
 | [default-features](#dependency-default-features) | No | bool | Require the features listed as on-by-default |
-| [features](#dependency-features) | No | string[] | The list of additional features to require |
+| [features](#dependency-features) | No | string\|[Feature Object][][] | The list of additional features to require |
```

Comment:
```suggestion
| [features](#dependency-features) | No | [Feature Object][][] | The list of additional features to require |
```

We should modify the definition of `Feature Object` to include that simple strings are valid and equivalent to:

```json
{ "name": "<feature name>" }
```